superglue/lib/reducers/index.ts

```diff
@@ -175,16 +199,14 @@ export function pageReducer(state = {}, action) {
     }
     case COPY_PAGE: {
       const nextState = { ...state }
-      const { from, to } = action.payload
+      const { from, to } = action.payload as CopyAction['payload']
```

Comment:
It's unfortunate that we have to cast the action payload so often here. I'm surprised Typescript can't tell the action type based on the `action.type`.

Reply:
Any chance you could do something like `case CopyAction['type']`: to help Typescript detect it better?

Reply:
Sadly that didn't work. Not sure what the right thing to do here is, but using Redux's `createAction` was a possibility, but i didn't want to do that yet.